tests/Libraries/Markdown/html_markdown_examples/footnote_invalid_duplicate.html

```diff
@@ -0,0 +1,32 @@
+<p class="osu-md__paragraph">
+First
+<sup id="osu-fnref-test">
+  <a class="osu-md__link osu-md__link--footnote-ref js-reference-link" href="#osu-fn-test" role="doc-noteref">1</a>
+</sup>
+</p>
+
+<p class="osu-md__paragraph">
+Second
+<sup id="osu-fnref-test__2">
+  <a class="osu-md__link osu-md__link--footnote-ref js-reference-link" href="#osu-fn-test" role="doc-noteref">1</a>
+</sup>
+</p>
+
+<li class="osu-md__list-item osu-md__list-item--footnote" id="osu-fn-test" role="doc-endnote">
```

Comment:
this extension sure likes producing weird output. This invalid list item aside, it also accepts `[^1]: > aaa` and produces blockquote. Similarly with `[^1]: 1. aaa` :neutral_face:

Reply:
Just do some *moderation* from wiki-side to not using weird syntax inside footnote list :upside_down_face: 

I think they only need to use emphasis and link for footnote list in wiki, cc: @TicClick for confirmation.

Reply:
I think so, yeah. maybe some other basic formatting like bold/underlined, but that's going to be very rare. typical footnote usage is link references or remarks like the one under ¹ at https://osu.ppy.sh/wiki/VCC#criteria